src/helm/proxy/clients/mechanical_turk_critique_exporter.py

```diff
@@ -57,13 +57,17 @@ def _render_template_crowd_html(task_template: CritiqueTaskTemplate) -> str:
     )
 
     instructions_crowd_html = f"<div>{_format_template_tags(task_template.instructions)}</div>"
-    questions_crowd_html = "\n".join([_render_question_crowd_html(question) for question in task_template.questions])
+    instruction_question_break_html = "<br><br><h4>Please answer the questions below:</h4>"
```

Comment:
I don't think we need so much whitespace here... if we really need it, we should use `style="margin-top: 2em"` instead.

Reply:
Discussed in person. Keep it as is for now.